galloper/api/project.py

```diff
@@ -90,7 +95,7 @@ def get(self, project_id: Optional[int] = None) -> Union[Tuple[dict, int], Tuple
                 projects = Project.query.filter(_filter).limit(limit_).offset(offset_).all()
             else:
                 projects = Project.query.limit(limit_).offset(offset_).all()
-        total = Project.query.order_by(Project.id.desc()).count()
+        total = self.__get_project_count(_filter)
```

Comment:
Fix calculation of total available projects
In other words:
If user assigned to 1 of 5 projects -> total for him should be equal to 1